cmd/helm/push.go

```diff
@@ -54,5 +54,9 @@ func newPushCmd(cfg *action.Configuration, out io.Writer) *cobra.Command {
 		},
 	}
 
+	f := cmd.Flags()
+	f.BoolVar(&client.InsecureSkipTLSverify, "insecure-skip-tls-verify", false, "skip tls certificate checks for the chart upload")
```

Comment:
Re naming convo in https://github.com/helm/helm/pull/10408/files#r802048813

This is a follow up: https://github.com/helm/helm/pull/10408#pullrequestreview-937164181 and https://github.com/helm/helm/pull/10408#issuecomment-1094282157

> @sabre1041 
> Functionality within this PR is good. However, I am concerned with the differentiation of terms between `helm registry login` and these subcommand.
>
> `--insecure` is used in `helm registry login` and `--insecure-skip-tls-verify` is used here. Why the differentiation? There should be a single way to specifying ignoring self signed certificates
>> @pytimer 
>> The helm pull use insecure-skip-tls-verify when use chart repository, so use this flag to be consistent in helm push/pull

Sounds like this is already inconsistent 🤔  I wonder if there is a good reason for that or if it was just an oversight. Can we pick a consistent flag?

Reply:
@scottrigby Would it make sense to incorporate `--insecure-skip-tls-verify` into the `helm registry login` command since it already has existing precedence in other commands?